bootstrap

```diff
@@ -121,6 +125,8 @@ while test $# != 0; do
     case "$1" in
     --prefix=*) dir=`arg "$1"`
                 prefix_dirs="$dir";;
+    --vcpkg-target-triplet=*)   vcpkg_target_triplet=`arg "$1"`
+                                vcpkg_target_triplet="-DVCPKG_TARGET_TRIPLET=${vcpkg_target_triplet}";;
```

Comment:
I believe we talked about using the name "base triplet" in the option name to remove ambiguity.

This option is useful, but should go in a separate PR. It will need to be deconflicted with the base triplet specification.